doc/architecture.rst

```diff
@@ -1506,11 +1506,11 @@ Ceph Clients
 
 Ceph Clients include a number of service interfaces. These include:
 
-- **Block Devices:** The :term:`Ceph Block Device` (a.k.a., RBD) service 
-  provides resizable, thin-provisioned block devices with snapshotting and
-  cloning. Ceph stripes a block device across the cluster for high
-  performance. Ceph supports both kernel objects (KO) and a QEMU hypervisor 
-  that uses ``librbd`` directly--avoiding the kernel object overhead for 
+- **Block Devices:** The :term:`Ceph Block Device` (a.k.a., RBD) service
+  provides resizable, thin-provisioned block devices that can be snapshotted
+  and cloned. Ceph stripes a block device across the cluster for high
+  performance. Ceph supports both kernel objects (KO) and a QEMU hypervisor
```

Comment:
I must to confess that if someone asked me what kernel objects are in relation Ceph or RBD (or, worse, used KO abbreviation), I wouldn't have the answer.  "Kernel objects" is very misleading technically because the _same_ RADOS objects can be accessed via the RBD kernel module or the RBD block driver in QEMU (and now also a similar driver in SPDK).  I would say something like:

> Ceph supports (provides?) a Linux kernel block device as well as a driver for the QEMU hypervisor that uses ``librbd`` directly -- avoiding the kernel round-trip overhead in virtualization use cases.

OTOH I checked and it appears that this kernel objects (KO) wording is over 10 years old!  Perhaps it's just me and people didn't feel confused enough to raise this...

Reply:
@zdover23 Suggest that in a future PR we change this to

>Ceph Block Devices can be mounted directly by a sufficiently recent Linux kernel (`KRBD`) by hypervisors including QEMU (`librbd`), and by userspace `rbd-nbd`.